.lando.yml

```diff
@@ -5,33 +5,55 @@ recipe: wordpress
 config:
   php: '7.2'
   via: nginx
-  webroot: trunk
+  webroot: public
   database: mariadb
   xdebug: false
 
+  conf:
+    server: config/wordpress.conf
+
 services:
   appserver:
     composer:
+      # Note installing phpunit should be temporary because core should really install it via composer on its own.
       phpunit/phpunit: "^6"
     install_dependencies_as_root:
       - apt-get update -y
       - apt-get install gnupg -y
-      - curl -sL https://deb.nodesource.com/setup_8.x | bash -
-      - apt-get install subversion -y
+      - curl -sL https://deb.nodesource.com/setup_10.x | bash -
       - apt-get install nodejs -y
-    install_dependencies_as_me_internal:
-      - mkdir ~/.npm-global
-      - npm config set prefix '~/.npm-global'
-      - npm install -g grunt-cli
-      - npm install -g gulp-cli
+      - apt-get install zip -y
+      - apt-get install subversion -y
+      # Allow loopback requests to work.
+      - echo "127.0.0.1  wordpressdev.lando.site" >> /etc/hosts
+    install_dependencies_as_me:
+      # Setup environment variables.
+      - if [ ! -e $LANDO_MOUNT/.env ]; then cp $LANDO_MOUNT/config/.env $LANDO_MOUNT/.env && sed -i "s#/app#$LANDO_MOUNT#" $LANDO_MOUNT/.env; fi
+    run:
+      # Setup WordPress for contributing via git.
+      - if [ ! -e $LANDO_MOUNT/public/trunk-git ]; then git clone --depth 1 https://github.com/WordPress/wordpress-develop.git $LANDO_MOUNT/public/trunk-git; fi
```

Comment:
I think this shouldn't be called `trunk-git` but rather `core-git`, since you could very well switch to `4.9` or other branches which are not `trunk` (and in Git it is `master` anyway).